### DISCUSSION TITLE
v2.5.0

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#2204] [General]
### New report: Insights report

This report allows you to pageviews and visitor data across multiple fields.

![image](https://github.com/umami-software/umami/assets/477975/d0db693c-a10a-4c0b-9813-acd59df0c0d6)

You can also filter down your results by adding additional filters.

![image](https://github.com/umami-software/umami/assets/477975/02ab2782-2166-4f96-9be5-5d59ce186e26)

### New report: Retention report

This report allows you to see how sticky your website is by showing how many users return after an initial visit.

![image](https://github.com/umami-software/umami/assets/477975/efb78548-13c2-4cf4-9e95-600cef10b47d)

### Top level navigation: Websites and Reports

We've added Websites and Reports as top level navigation items. On the websites page, you can view of your own websites as well as team websites.

![image](https://github.com/umami-software/umami/assets/477975/2560898d-42cc-4c62-b376-0ce96eff6ff4)

On the reports page, you can view all the reports created across all the websites you have access to:

![image](https://github.com/umami-software/umami/assets/477975/85ec6200-669c-427d-9071-cfa9a93c21d2)

### Search and pagination

Tables now have a search box so you can perform searches across your websites, teams, and users. We also added pagination that will show 10 records at a time so you don't like to scroll forever.

## Changes

- Language translation updates.
- Created additional database indexes that should help speed up your queries.
- Cloudflare headers for region and city information are now supported.

A huge thanks to all the contributors on this release! @ImgBotApp @essesoul @zer0eXploit @HtetOoWaiYan

<hr /><em>This discussion was created from the release <a href='https://github.com/umami-software/umami/releases/tag/v2.5.0'>v2.5.0</a>.</em>

**Comment by 4ortytwo** [comment]
Updated to 2.5.0 just now and getting this screen. Deployed on DO, using Dokku. Reverting back to 2.4.1 doesn't seem to help. 
<img width="861" alt="image" src="https://github.com/umami-software/umami/assets/44406493/d04a68a6-c769-444d-af58-2ae3314b8376">

**Reply by 4ortytwo** [reply]
To add to this, getting this when I try to rebuild 
<img width="1034" alt="image" src="https://github.com/umami-software/umami/assets/44406493/6d95a080-f908-49a7-8d14-446d2dbd720f">
